content/es/docs/reference/glossary/kube-apiserver.md

```diff
@@ -0,0 +1,25 @@
+---
+title: API server
+id: kube-apiserver
+date: 2020-07-01
+full_link: /docs/reference/generated/kube-apiserver/
+short_description: >
+  Componente del plano de control que expone la API de Kubernetes.
+
+aka:
+- kube-apiserver
+tags:
+- architecture
+- fundamental
+---
+
+El API server es el componente del {{< glossary_tooltip text="plano de control " term_id="control-plane" >}}
```

Comment:
```suggestion
El API server es el componente del {{< glossary_tooltip text="plano de control" term_id="control-plane" >}}
```

Reply:
Fixed in https://github.com/kubernetes/website/pull/22259/commits/c2838ec5285df2da076ee458f7d93a6ef991ecc4